src/targets/CNC/FluidNC/TargetContext.js

```diff
@@ -41,7 +41,11 @@ const useTargetContext = () => useContext(TargetContext);
 const useTargetContextFn = {};
 
 const TargetContextProvider = ({ children }) => {
-  const [positions, setPositions] = useState({});
+  const [positions, setPositions] = useState({
+    x: "0.00",
+    y: "0.00",
+    z: "0.00",
+  });
 
```

Comment:
I think put 0.00 is leading to user error if position is not known - I  put char `?` instead only for X to let the dynamic detection of Axis number in status output